examples/basic/settings.js.dist

```diff
@@ -3,3 +3,7 @@ export const keycloakConfig = {
     realm: "YOUR-REALM",
     clientId: "YOUR-CLIENT-ID",
 };
+
+export const uhliveConfig = {
+  url: "wss://api.uh.live",
+}
```

Comment:
I think we should better remove the `url: uhliveConfig.url` line in `main.js` (for both basic and advanced examples), because the production URL in the default one, there is no need for this property except in dev/staging.

Reply:
I will update the advanced folder in another PR